### ISSUE TITLE
Check icon overlaps the menu item icon when toggled

### ISSUE THREAD

**Issue by Hanksha** [eclipse-theia/theia#8816]
### Bug Description:

For menu actions which use a command with a toggled state the check icon is overlapping the menu item icon when rendered. 
See screenshot:

![image](https://user-images.githubusercontent.com/6960133/100901456-1e35aa80-34c4-11eb-8250-c79ed4773352.png)

I was expecting something like this:
![image](https://user-images.githubusercontent.com/6960133/100901891-a1ef9700-34c4-11eb-9bf4-c50084242d9e.png)

But maybe I'm doing something wrong with the `iconClass`.

### Steps to Reproduce:

1. Create a `Command` with an `iconClass` and register it.
2. Register a `CommandHandler` for that command which implements `isToggled`.
3. Register a menu action for that command.

### Additional Information

- Operating System: macOS
- Theia Version: 1.5.0

**Comment by vince-fugnitto** [comment]
@Hanksha I don't believe we support having `icons` and `toggled` state simultaneously for menu items in the framework (I assume that is why you had to update `css` with `!important` rules). I'm not sure this is a bug as you mentioned and has to do with the `css`.

**Comment by Hanksha** [comment]
@vince-fugnitto No the `!important` rules were for something else because it's the first time I deal with a menu which has toggled items. I didn't know it was not supported, hence the bug. Is this something we could support or is that a limitation of phosphorjs menus?

**Comment by vince-fugnitto** [comment]
> @vince-fugnitto No the `!important` rules were for something else because it's the first time I deal with a menu which has toggled items. I didn't know it was not supported, hence the bug.

@Hanksha I was confused by the first screenshot, is this a theia-based application?

> Is this something we could support or is that a limitation of phosphorjs menus?

I have not investigated yet.

**Comment by Hanksha** [comment]
> @Hanksha I was confused by the first screenshot, is this a theia-based application?

@vince-fugnitto Yes it is, I'm using the `ContextMenuRenderer` to render the context menu in my custom toolbar.

> I have not investigated yet.
Alright, I'll look into it as well.

**Comment by Hanksha** [comment]
I was able to reproduce it with phosphorjs menus so I think it would have to be fixed there.
